pontoon/teams/static/js/permissions.js

```diff
@@ -100,6 +100,12 @@ $(function() {
     var clone = $(this).remove();
 
     target.find('ul').prepend(clone.removeClass('hover'));
+
+    // After every change sort translators/managers by email
+    target.find('ul li').sort(function (a, b) {
+      return $(a).text().localeCompare($(b).text());
+    }).appendTo(target.find('ul'));
+
```

Comment:
I'd rather remove this actually.

That way the newly added user is always on top of the list and stands out. Otherwise, if the list is long, it might get lost. It's also consistent with how locale selector behaves in the Admin.

Reply:
Totally makes sense. Actually I even remember our first discussion about this behaviour.
My fault, will redo!